blackbox/docs/general/ddl/storage.txt

```diff
@@ -0,0 +1,52 @@
+=========
+ Storage
+=========
+
+How data is stored can be tuned on each column similar to how indexing is defined.
+
+Column Store
+============
+
+Beside of storing the row data as-is (and indexing each value if wanted), each
+value term is stored into a `Column Store`_ by default. The usage of a `Column
+Store`_ is greatly improving global aggregations and groupings and enables
+ordering possiblity as the data for one column is packed at one place. Using the
+`Column Store`_ limits the values of :ref:`data-type-string` columns to a
+maximal length of 32766 bytes.
+
+Turning the `Column Store`_ off will remove the length limitation but also will
+disable ordering functionallity.
+
+Example:
+::
+
+    cr> CREATE TABLE t1 (
+    ...   id INTEGER,
+    ...   url STRING STORAGE (columnstore = false)
+    ... );
+    CREATE OK, 1 row affected  (... sec)
+
+Doing so will enable support for inserting strings longer than 32766 bytes into
+the ``url`` column, but will also disable the possibility to sort on the ``url``
```

Comment:
Is it true? We still can sort in memory, right?

Reply:
as already discussed at previous comment by @mfussenegger , we could. pls read discussion

Reply:
@seut, sorry missed that.